src/it/java/net/spy/memcached/CancelFailureModeTest.java

```diff
@@ -101,7 +101,7 @@ public FailureMode getFailureMode() {
 
   @Override
   protected void flushPause() throws InterruptedException {
-    Thread.sleep(100);
+    Thread.sleep(10000);
```

Comment:
Any problem with the previous number?

Reply:
Yes. Sometimes it passes, sometimes it failed. I think it is because of longer time to initializing client in TLS mode than non-TLS.